src/styles/blocks/card.scss

```diff
@@ -0,0 +1,106 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid $borderColor;
+  border-radius: $borderRadius;
+  width: $cardWidth;
+  padding: 32px 16px 16px;
+  transition: transform 300ms ease;
+
+  &__image {
+    width: $imageWidth;
+    height: $imageHeight;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-family: $bodyFont;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 0 0 4px;
+    color: $titleColor;
+  }
+
+  &:hover {
+    transform: scale(1.2);
+
+    .card__title {
+      color: #34568b;
+    }
+  }
+
+  &__description {
+    font-family: $bodyFont;
+    font-size: 10px;
+    line-height: 14px;
+    margin-top: 0;
+    margin-bottom: 16px;
+    color: $descriptionColor;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 24px;
+    font-size: 10px;
+
+    &-numbers {
+      font-family: $bodyFont;
+      font-size: 10px;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+
+    &-title {
+      font-size: 12px;
+      line-height: 18px;
+      color: $priceTitleColor;
+    }
+
+    &-value {
+      font-family: $bodyFont;
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+      color: $priceNumberColor;
+    }
+  }
+
+  &__button {
+    font-family: $bodyFont;
+    font-weight: 700;
+    font-size: 14px;
+    text-transform: uppercase;
+
+    line-height: 16px;
+    height: $buttonHeight;
+    width: 100%;
+
+    border: 1px solid $buttonBorderColor;
+    border-radius: $buttonBorderRadius;
+
+    background-color: $buttonBackgroundColor;
+    color: $buttonTextColor;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    text-decoration: none;
+    
+    transition: 300ms;
```

Comment:
It's better to specify to what properties you need to apply the transition.

![image](https://github.com/mate-academy/layout_catalog/assets/54860641/d13f444e-49d7-42fa-acdd-7225c2ee1da0)

Reply:
got it, thanks for these suggestions. Need to pay a more attention with the documentation.